app/Http/Controllers/RecruitingController.php

```diff
@@ -146,11 +146,7 @@ public function doThreadCheck(Request $request)
     public function validateMemberId($member_id)
     {
         if (app()->environment() === 'local') {
-            if ($member_id === 31832) {
```

Comment:
Was there any reason to limit recruitment to this member ID in a local environment? I removed it so the recruitment flow is easier to test locally.

Reply:
That's my id, back when it was just me. Fine to remove